pkg/controller/provider/container/ocp/collection.go

```diff
@@ -10,6 +10,7 @@ import (
 	core "k8s.io/api/core/v1"
 	storage "k8s.io/api/storage/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	cnv "kubevirt.io/client-go/api/v1"
```

Comment:
The upstream project is `kubevirt`, not `cnv`. So, we shouldn't see any `cnv` reference in our code.